lib/sqlalchemy/ext/asyncio/session.py

```diff
@@ -336,7 +340,10 @@ async def refresh(
         )
 
     async def run_sync(
```

Comment:
I guess it's not easy to add a typing test for this.

Maybe an expected error could be added, like passing a parameter foo=1 to a function that accepts a bar parameter?

You can probably add a couple in the files in `test/typing/plain_files/ext/asyncio`, using this form to expect the error:
https://github.com/sqlalchemy/sqlalchemy/blob/5c88498ba8ba5f7c524d5aca130e5a59a8940766/test/typing/plain_files/sql/sql_operations.py#L47-L48

Reply:
Thanks for your help, it worked.

I've added missing tests for the `AsyncConnection.run_sync` method too.